concourse/tasks/terraform.sh

```diff
@@ -21,6 +29,6 @@ cd "${DEPLOYMENT_PATH}"
 
 terraform init -input=false -backend-config "${ENVIRONMENT}.backend"
 
-terraform apply -input=false \
+terraform "$tf_action" -input=false \
   -var-file "../variables/${ENVIRONMENT}/common.tfvars" \
-  -auto-approve
+  -auto-approve "$@"
```

Comment:
Will this work for plan? Will you not get an error because of the `-auto-approve` flag not defined?

Reply:
Good catch, thanks. I should have tested both cases. Fixed.

\<rant> It's so frustrating that Terraform doesn't have a consistent way to disable interactive prompting 😭 \</rant>

Reply:
Also fixed a couple of shellcheck lints.